.travis.yml

```diff
@@ -3,12 +3,11 @@ node_js: 12.18.4
 addons:
   apt:
     update: true
+  chrome: stable
```

Comment:
why this is needed ?

Reply:
It was just moved from bottom to top. It is needed for integrational tests. I just fixed `.travis.yml` according to the travis documentation.

Reply:
though the integration tests works ok without this

Reply:
@orenyodfat no! It was moved from bottom section. Please, see whole file to understand